### PR TITLE
docs : Rephrased the alert note.

### LINKED ISSUE #4528: 'Adding assets and images' page issue
Page URL: https://flutter.dev/docs/development/ui/assets-and-images.html
Page source: https://github.com/flutter/website/tree/master/src/docs/development/ui/assets-and-images.md

Description of issue:

The description is inconsistent. **Either**
"Note: Only files located directly in the directory are included. To add files located in subdirectories, create an entry per directory."
is true **or**
"Then both graphics/background.png and graphics/dark/background.png are included in your asset bundle."
is true
**OR** there is some quite different handling in the background?!?
Sorry, I cannot fix it by myself since this handling is still not completely clear for me.

Regards
Hajo

### PR DESCRIPTION
**Pull request by aromalanil**
Rephrased the alert note to a more accurate and clear note. This will fix #4528

### CODE REVIEW

File: src/docs/development/ui/assets-and-images.md [hunk 1]
```diff
@@ -38,7 +38,10 @@ flutter:
 ```
 
 {{site.alert.note}}
-  Only files located directly in the directory are included unless there are files with the same name inside a subdirectory (see Asset Variants). To add files located in subdirectories, create an entry per directory.
+  Only files located directly in the directory are included unless
+  there are files with the same name inside a subdirectory 
+  (see Asset Variants). To add files located in subdirectories, create
```
Comment (aromalanil) [inline src/docs/development/ui/assets-and-images.md:43]: Should I add a link to Asset Variants topic here?

Comment (sfshaza2) [reply src/docs/development/ui/assets-and-images.md:43]: Yes please! And thanks for breaking up that massive line!

Comment (aromalanil) [reply src/docs/development/ui/assets-and-images.md]: I have added the link

**Review by sfshaza2 — APPROVED** [review]
LGTM

**Pull request merged**